File: tuts/011-getting-started-batch-fargate/php/GettingStartedScenario.php
<?php
// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
// SPDX-License-Identifier: Apache-2.0

require 'vendor/autoload.php';

use Aws\Batch\BatchClient;

$client = new BatchClient(['region' => 'us-east-1']);
echo "Running Batch getting started scenario...\n";
// TODO: setup, interact, teardown
echo "Scenario complete.\n";

File: tuts/011-getting-started-batch-fargate/swift/Sources/GettingStartedScenario.swift
// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
// SPDX-License-Identifier: Apache-2.0

import AWSBatch

@main
struct GettingStartedScenario {
    static func main() async throws {
        let client = try BatchClient(region: "us-east-1")
        let wrapper = BatchWrapper(client: client)
        print("Running Batch getting started scenario...")
        // TODO: setup, interact, teardown
        let _ = wrapper
    }
}
